agent/src/main/kotlin/org/jetbrains/teamcity/vault/agent/VaultBuildFeature.kt

```diff
@@ -142,16 +151,17 @@ class VaultBuildFeature(dispatcher: EventDispatcher<AgentLifeCycleListener>,
 
     private fun createAwsIamAuthentication(restTemplate: RestTemplate): AwsIamAuthentication {
         val options = AwsIamAuthenticationOptions.builder()
-                .credentialsProvider(InstanceProfileCredentialsProvider.getInstance()).build()
+            .credentialsProvider(InstanceProfileCredentialsProvider.getInstance())
```

Comment:
Was that done intentionally or by auto code-formatter? 
Similar changes don't help later when using `git blame`

Reply:
yep, auto formatter + some reasonable (imho) applied intentions.
reverted